### ISSUE TITLE
Please design generic alerts.

### ISSUE THREAD

**Issue by rmisio** [OpenBazaar/openbazaar-desktop#231]
Given the new design of overlays (button outside of main content, etc...), can we please have a design for generic alerts?

![image](https://cloud.githubusercontent.com/assets/794517/21329413/65a61b74-c5ed-11e6-8344-e4d91eda4336.png)

For example, would it be as simple as moving the close button outside of the main content, like so?

![image](https://cloud.githubusercontent.com/assets/794517/21329433/8178f222-c5ed-11e6-831c-511fe865167e.png)

I'm assuming we should continue to vertically center small alerts?

**Comment by morebrownies** [comment]
Yeah, I think we'll need to move the close button out and keep it vertically aligned. If you want to make those 2 changes @rmisio, I can go in after and touch up padding and smaller details.
